docs/src/main/sphinx/release/release-470.md

```diff
@@ -0,0 +1,105 @@
+# Release 470 (5 Feb 2025)
+
+## General
+
+* Add [](/connector/duckdb). ({issue}`18031`)
+* Add [](/connector/loki). ({issue}`23053`)
+* Add support for `WITH SESSION` to set session properties with `SELECT`
+  queries. ({issue}`24889`)
+* Add support for access control systems to control privileges for
+  `CREATE FUNCTION` and `DROP FUNCTION`. ({issue}`24696`)
+* Improve compatibility of fault-tolerant exchange storage with S3-compliant
+  object stores. ({issue}`24822`)
+* Improve compatibility of fault-tolerant exchange storage with
+  Hadoop-compatible file systems and the HDFS as exchange type by adding the
+  `exchange.hdfs.skip-directory-scheme-validation` property. ({issue}`24627`)
+* Export JMX metric for `blockedQueries`. ({issue}`24907`)
+* {{breaking}} Remove the deprecated precomputed hash optimizer and the related
+  `optimize_hash_generation` configuration property. ({issue}`24792`)
+* Fix failure when using upper-case variable names in SQL user-defined
+  functions. ({issue}`24460`)
+* Prevent failures of the `ARRAY_HISTOGRAM` function resulting from null values.
+  ({issue}`24765`)
+* Prevent wrong JMX metrics values when using dynamic catalogs. ({issue}`24908`)
+
+## JDBC driver
+
+* {{breaking}} Raise minimum runtime requirement to Java 11. ({issue}`23639`)
+
+## CLI
+
+* {{breaking}} Raise minimum runtime requirement to Java 11. ({issue}`23639`)
+
+## Delta Lake connector
+
+* Prevent connection leakage when using the Azure file system. ({issue}`24116`)
+* Deprecate use of the legacy file system support for Azure Storage, Google
+  Cloud Storage, IBM Cloud Object Storage, S3 and S3-compatible object storage
+  systems. Use the migration guides for [Azure
+  Storage](fs-legacy-azure-migration), [Google Cloud
+  Storage](fs-legacy-gcs-migration), and [S3](fs-legacy-s3-migration) to assist
+  if you have not switched from legacy support. ({issue}`24878`)
+* Fix incorrect behavior of the `vacuum` procedure deleting unexpected files
```

Comment:
Run-on sentence. How about: "Fix incorrect behavior of `vacuum`, which would delete unexpected files when their path contains special characters".

Is this an case of table corruption? If so, maybe rephrase in those terms? E.g., "Fix potential table corruption when running `vacuum` on tables that contain special characters in their path" (not sure what exactly needs to have special characters for the issue to occur, though)